pkg/virt-controller/watch/vmi.go

```diff
@@ -437,10 +499,22 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 
 	c.syncReadyConditionFromPod(vmiCopy, pod)
 	if vmiPodExists {
-		err := c.syncPausedConditionToPod(vmiCopy, pod)
-		if err != nil {
+		var foundImage string
+		for _, container := range pod.Spec.Containers {
+			if container.Name == "compute" {
+				foundImage = container.Image
+				break
+			}
+		}
+		vmiCopy = c.setLauncherContainerInfo(vmiCopy, foundImage)
```

Comment:
fyi, i moved this code because it influences one of the labels that syncs to the pod.